docs/configuration/jobs.md

```diff
@@ -78,13 +78,43 @@ jobs:
   branch: main
 ```
 
-Note that in case of more complex scenarios, it can be useful to create also
-a template job, that should not be triggered in Packit at all.
-This is typically useful when creating a number of tests which needs various
-parameters, but many parameters stay same. In such a case set the 
-`trigger` to `ignore` in the template job and use the expected trigger in real jobs.
+Do not forget that you can also use the top-level keys to store common keys like
+`targets` and `actions` and only override them as needed in the `jobs` or
+`packages` section.
```

Comment:
I think I know what you want to say here, but it's not doing a great job. You can't use top-level keys (except for the `_` that you allowed in the other PR), because we validate the schema.

Reply:
I'm trying to say that instead of having a 
```yaml
jobs:
  - &copr_build
    job: copr_build
    trigger: pull_request
    targets: &targets
      - fedora-all
      - epel-all
  - &tests
    job: tests
    trigger: pull_request
    targets: *targets
```
we should remember that we can define `targets` at the top-level
```yaml
targets:
  - fedora-all
  - epel-all
jobs:
  - &copr_build
    job: copr_build
    trigger: pull_request
  - &tests
    job: tests
    trigger: pull_request
```

That would be allowed by the schema validation. Any ideas on how to rephrase it?

Reply:
```suggestion
Do not forget that you can also define common Packit config options at the
top level (such as `targets` and `actions`) and only override them when
needed in the `jobs` or `packages` section.
```

maybe?

Reply:
Sounds good to me. I'll leave you to apply it if you have some other edits you want to stack up.